lxd/storage/backend_lxd.go

```diff
@@ -2677,6 +2677,50 @@ func (b *lxdBackend) MountInstance(inst instance.Instance, op *operations.Operat
 	return mountInfo, nil
 }
 
+// DelegateInstance sets up an instance volume for delegation.
+func (b *lxdBackend) DelegateInstance(inst instance.Instance) error {
+	// Delegation only works for running instances.
+	if inst.InitPID() < 1 {
```

Comment:
Is there a significant benefit using `inst.InitPID` instead of `inst.IsRunning`?

Reply:
That's reuse from code above.

I believe it was intentional to also catch things like the freezing state, we have similar checks in a few other places.